src/components/User/ProfileDetails/CardDetailsList.js

```diff
@@ -7,6 +7,8 @@ import { Typography, withStyles } from '@material-ui/core';
 const styles = theme => ({
   title: {
     marginTop: theme.spacing.unit * 2,
+    color: 'gray',
+    fontSize: '30px',
```

Comment:
if you going to pass some value using `px` unit you can just pass a number

```suggestion
    fontSize: 30,
```